lighthouse-core/test/lib/i18n/locales-test.js

```diff
@@ -19,6 +19,7 @@ describe('locales', () => {
       in: 'id',
       iw: 'he',
       mo: 'ro',
+      tl: 'fil',
```

Comment:
in node 16, the updated ICU version gives `'fil'` for `Intl.getCanonicalLocales('tl')` ([as anticipated](https://github.com/GoogleChrome/lighthouse/blob/545176a04fa51a68ce250a47ecbb9b9be1367c7c/lighthouse-core/lib/i18n/locales.js#L105)), so need to add it here.